packages/ui/src/domains/LandingPageBuilder/LandingPageBuilder.tsx

```
@@ -0,0 +1,2509 @@
+	height,
+	opacity,
+	objectFit,
+	onUpdate,
+}: {
+	src: string;
+	alt: string;
+	width: number;
+	height: number;
+	opacity: number;
+	objectFit: string;
+	onUpdate: (newProps: any) => void;
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)